### PR TITLE
Add readmes from the main codebase

### PR DESCRIPTION
**Pull request by Jesse-NH**
Adds README files from https://github.com/NUbots/NUbots into pages using a Python script.

## Preview
<https://deploy-preview-308--nubook.netlify.app/system/modules/actuation/>

### CODE REVIEW

**Review by ysims — COMMENTED** [review]
- make folder names (Module, Nuclear, etc) lowercase
- revert changes to the tool icon
- rename md files to mdx for consistency

File: src/components/sidebar/sidebar.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const iconMap = {
   Team: IconTeam,
   System: IconSystem,
   Guides: IconGuides,
+  Readmes: IconGuides,
```
Comment (ysims) [inline src/components/sidebar/sidebar.jsx:16]:
```suggestion
  Readmes: IconReadMes,
```

**Comment by ysims** [comment]
I'd also add in a brief description for each page, something like "Readmes for the localisation modules in the main codebase".

**Review by LachlanCourt — APPROVED** [review]
Looks good just need to run `yarn format` to fix the build :)

**Comment by Tom0Brien** [comment]
I feel like there is a better title for this over README's, like `Code documentation` or something along those lines

**Comment by ysims** [comment]
> I feel like there is a better title for this over README's, like `Code documentation` or something along those lines

Code documentation is a bit vague, but I guess what these pages really are, are the descriptions of each module in the codebase. The section could just be 'Modules'. Alternatively it could also live in System and be a chapter called 'Modules', but then we really need to fix up navigation and the sidebar. 

I also need to fix up the icons in this PR.

**Comment by Tom0Brien** [comment]
> > I feel like there is a better title for this over README's, like `Code documentation` or something along those lines
> 
> Code documentation is a bit vague, but I guess what these pages really are, are the descriptions of each module in the codebase. The section could just be 'Modules'. Alternatively it could also live in System and be a chapter called 'Modules', but then we really need to fix up navigation and the sidebar.
> 
> I also need to fix up the icons in this PR.

Having it live in System is a good idea. I am also still very much in favour of moving `NUbook` into `NUbots` repo to streamline documentation PR's. We could have this all work more seamlessly that way

**Review by A-Herfel — APPROVED** [review]
Looks good. The NUbook installation instructions are out of date because Node 16 is so old that Node no longer gives instructions on how to install it :)

**Comment by ysims** [comment]
> Looks good. The NUbook installation instructions are out of date because Node 16 is so old that Node no longer gives instructions on how to install it :)

Yeah I noticed they were out of date the other day - would you be up for fixing that page up?
For PRs though, there is no need to build NUbook yourself - just use the live preview link in the description.

**Pull request merged**